akka-http/build.sbt

```diff
@@ -1,11 +1,9 @@
 import akka._
 
-AkkaBuild.defaultSettings
-AkkaBuild.experimentalSettings
-Formatting.formatSettings
-OSGi.http
+//OSGi.http
```

Comment:
We probably need this right @2m?
As it defines the exports. I can work on that next